src/app/passengers/components/smart/trip-planner/trip-planner.component.html

```diff
@@ -1,25 +1,23 @@
 <mat-card>
   <h1>{{ title }}</h1>
   <div class="container">
-    <agm-map [latitude]="latitude" [longitude]="longitude" [zoom]="zoom">
+    <agm-map [latitude]="latitude" [longitude]="longitude" [zoom]="zoom" (mapClick)="getGeoCode($event)">
       <div appDirectionsService></div>
     </agm-map>
   </div>
-  <div class="inprogress">
+  <!-- <div class="inprogress">
```

Comment:
Do we need this commented out code?

Reply:
this has been addressed & removed on my recent push just made now